components/Notifications.tsx

```diff
@@ -115,22 +116,20 @@ function Notifications() {
                           {message.body.split("")}
                         </AlertDescription>
                       </Flex>
-                      <Avatar src={message.icon} />
-                      <CloseButton
-                        alignSelf="flex-start"
-                        position="relative"
-                        rounded="full"
-                        right={-1}
-                        top={-1}
-                        onClick={async (e) => {
-                          e.preventDefault();
-                          deleteMessage(id);
-                        }}
-                      />
+                      <Avatar src={types?.[message.type]?.imageUrls.md} />
                     </Alert>
                   ))
               )}
+	      <Flex justifyContent="center">
+	        <GmButton
+                  leftIcon={<NotificationsIcon isDisabled={!isSubscribed} />}
```

Comment:
All these buttons should also be disabled when unsubscribing, or else you can click the other button real quick and get a bad state. Perhaps we should update the hooks such that `isSubscribed` is set to `false` immediately after starting the unsubscribe?